test/visualTest.ts

```diff
@@ -1120,42 +1122,50 @@ describe("EnhancedScatterChart", () => {
         });
     });
 
-    describe("Telemetry", () => {
-        beforeEach(() => {
-            dataView.metadata.objects = {
-                backdrop: {
-                    show: true
-                }
-            };
-        });
+    describe("URL link", () => {
+        // beforeEach(() => {
```

Comment:
why commented?

Reply:
with rewritten telemetry service and tests, there is no need to simulate backdrop enable cases, because we now test only url links with containsExternalURL method

Reply:
remove redundant comments